src/content/docs/apm/agents/manage-apm-agents/agent-data/triage-run-diagnostics.mdx

```diff
@@ -0,0 +1,192 @@
+
+You know that checkout is failing in your app but you're unsure of what's causing the actual error. In order to figure out what part of the checkout process is breaking you need to investigate your apps `Transactions`. A transaction represents the “work” that happens from when your application receives a request to when it sends a response. In order to determine which transaction is breaking, click on the **Transactions** tab.
+
+A slow transaction can be a strong indicator that something is behaving abnormally, so digging deeper into your slowest transaction is a good place to start. Once you click into **Transactions** you will see a list of all the "work" that is happening behind the scenes for your `checkout-service` application. On the left-hand side you will see which transaction is taking the most time. For the purpose of this example, the slowest transaction is `Sinatra/Rubytron/POST #purchaseCart.` Click on the transaction.
+
+<img
+    title="transactions tab in APM summary "
+    alt="A gif showing you how to dive deep into transactions"
+    src={apmTransactionsTabinApmOverview}
+/>
+
+</Step>
+
+<Step>
+### Dig deeper into your slowest transaction [#slow-transactions]
+
+Each transaction is comprised of many different units of "work." The **Transactions detail** page provides a holistic view of a single transaction by breaking it down into individual pieces of "work". The histogram on top visually demonstrates which pieces of the transaction are responding slower than others. 
+
+For the `Sinatra/Rubytron/POST #purchaseCart` transaction there are many different areas where the transaction could be failing: databases, servers, controllers, middleware and more. You can toggle between each different segment of the transaction to identify trends and spikes.
+
+In this case, the controller, `Sinatra/Rubytron/POST` is the slowest segment and is taking 59.83% of the transactions total time. 
+
+    <img
+    title="Slowest transaction"
+    alt="A gif that demonstrates how to select the slowest transaction"
+    src={accountsSelectSlowestTransaction}
+/>
+
+</Step>
+
+<Step>
+### Take a look at throughput 
+
+While looking at response time for this transaction, you can also investigate throughput. Throughput is a way to measure the amount of work your application is handling. 
+
+Since you're experiencing a spike in errors in the `checkout-service` application and the `Sinatra/Rubytron/POST #purchaseCart` transaction, it's helpful to compare those errors with your throughput results. 
+
+If your throughput is very high during a time of service degradation it could indicate that your application is processing more work than it can handle. On the other hand, low throughput could indicate that your app isn't handling very many requests. This could simply mean that it's not used very much or it could mean that a service upstream that calls to your application is broken and requests aren't being processed. 
```

Comment:
Missing comma after "degradation" I think for the if statement

Reply:
What does this throughput chart tell us?

Reply:
thank you both! fixed.